src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java

```diff
@@ -782,7 +812,7 @@ private Set<String> convertUserInput(String input) {
 			}
 			return Collections.emptySet();
 		} catch (Exception e) {
-			logger.error("In valid adapter properties input");
+			logger.error(String.format("Invalid adapter properties input: %s", e.getMessage()));
```

Comment:
please log e in exception :  catch (Exception e) {
			logger.error(String.format("Invalid adapter properties input: %s", e.getMessage()), e)